docs/.vuepress/components/AlertDialogDoc.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <StackLayout>
+        <Button text="Show Alert Dialog" width="200" @tap="showAlertDialog" id="alertDialog_docs" />
+        <div class="flex m-t-10">
```

Comment:
try to use our components instead of divs. flexboxlayout can be used here.

Reply:
> try to use our components instead of divs. flexboxlayout can be used here.

Replaced with FlexboxLayout.